CMakeLists.txt

```diff
@@ -7,6 +7,12 @@ set(CMAKE_INSTALL_LIBDIR "/nowhere")
 include(GNUInstallDirs)
 include(FindPythonModule)
 
+if ( "$ENV{PYTHONPATH}" STREQUAL "" )
+    set(ENV{PYTHONPATH} "${PROJECT_SOURCE_DIR}:${PROJECT_SOURCE_DIR}/shared:${PROJECT_SOURCE_DIR}/shared/modules")
```

Comment:
Why to set `PYTHONPATH` to anything else than project source dir? There is that testoval module, but why does that one deserve its own directory?

Reply:
@matejak `testoval` module is being moved in the next PR. I can move it to this PR if you want though, and then we can clean up CMake early. I agree that this PR is large though :) Preference? 

Perhaps remove `pep8` fixes but add moving `testoval`?

Reply:
@matejak Moved into this PR, updated CMakeLists.txt in the last commit.